go/client/cmd_simplefs_close.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"encoding/hex"
+	"errors"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+// CmdSimpleFSClose is the 'simplefs srar' command.
+type CmdSimpleFSClose struct {
+	libkb.Contextified
+	opid keybase1.OpID
+}
+
+// NewCmdSimpleFSClose creates a new cli.Command.
+func NewCmdSimpleFSClose(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "close",
```

Comment:
I don't think the concept of an "open file" should be exported to the user.  In the case of `read`ing a file out for streaming, control-C'ing the process should close the file.  Maybe we also need a cleanup on the service-side for this after a while too.